mart/attack/adversary.py

```diff
@@ -200,6 +213,11 @@ def attacker(self):
 
         self._attacker = self._attacker(accelerator=accelerator, devices=devices)
 
+        # Remove recursive adversarial training callback from lightning.pytorch.callbacks_factory.
+        for callback in self._attacker.callbacks:
+            if isinstance(callback, AdversarialTraining):
+                self._attacker.callbacks.remove(callback)
+
```

Comment:
Is this actually necessary in this PR? It seems like this PR does not contain logic to use callback entry point? I would rather have this logic in that PR.